torch/autograd/functional.py

```diff
@@ -472,6 +544,13 @@ def jacobian(func, inputs, create_graph=False, strict=False, vectorize=False):
          tensor([[3., 0.],
                  [0., 3.]]))
     """
+    if forward_ad:
+        if create_graph:
+            raise RuntimeError('torch.autograd.functional.jacobian: `create_graph=True` '
```

Comment:
Why is this one not supported? Just because we don't test it?
Also should this be a NotImplementedError ?

Reply:
Yeah I haven't tested it. Just putting NotImplementedError for now